package.json

```diff
@@ -42,8 +42,10 @@
     "react-dom": "15.6.1",
     "react-helmet": "5.1.3",
     "react-redux": "5.0.5",
-    "redux": "3.7.2",
+    "redux": "3.7.1",
+    "redux-form": "^7.0.0",
     "remote-redux-devtools": "0.5.12",
+    "shortid": "^2.2.8",
```

Comment:
Ok I got this I think. https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md